docs/main_text.md

```diff
@@ -214,5 +214,17 @@ TODO: screenshots? plots that show effect of different scaling options?
 
 # Species compendia
 
+TODO: Note about release schedule and/or presence on Zenodo
+
+refine.bio periodically releases compendia comprised of all samples from a species that were run on one of the platforms we support.
+We refer to these as **species compendia**. 
+We process these compendia in a manner that is different from the options that are available via the web user interface. 
+Instead of selecting only genes available in all samples, we take the union of all genes, filling in any missing values with `NA` (e.g., perform a full outer join).
+We drop any genes that have missing values in greater than 30% of samples.
+We impute the remaining missing values with KNN impute.
+We then quantile normalize all samples as described above.
+
+TODO: More information about QN, Illustration for full outer join, since we'll likely have one about the inner join above
+
 
```

Comment:
Does it make sense to link a list of all the previous compendia releases here? I know we don't have any but when we do.

Reply:
If we use a research data repository like [figshare](https://figshare.com) or [Zenodo](https://zenodo.org), they will support versioning so I think linking to the current version may be sufficient because all previous versions will be listed on the entry.

Reply:
I agree that we should use a repository that supports versioning 👍